app/src/test/java/org/simple/clinic/scanid/ShortCodeInputTest.kt

```diff
@@ -0,0 +1,44 @@
+package org.simple.clinic.scanid
+
+import com.google.common.truth.Truth.assertThat
+import junitparams.JUnitParamsRunner
+import junitparams.Parameters
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(JUnitParamsRunner::class)
+class ShortCodeInputTest {
+
+  @Parameters(value = [
+    "3",
+    "34",
+    "345",
+    "3456",
+    "34567",
+    "345678",
+    "34567890"
```

Comment:
The hint we have defined earlier for the short code was `123 4567`, which means that the user might enter the short code with a space in between. Are we going to handle this case elsewhere?

Reply:
Raised a story for this on Pivotal.
https://www.pivotaltracker.com/story/show/167992229